extra/points-of-interest/markers.json

```diff
@@ -4899,13 +4920,6 @@
 		"y": 32821,
 		"z": 7
 	},
-	{
-		"description": "",
-		"icon": "red down",
-		"x": 32201,
-		"y": 32948,
-		"z": 7
-	},
```

Comment:
Is this removal intentional?

Reply:
It isn't, however not sure where it came from as I did not remove any markers